src/serializers.ts

```diff
@@ -46,3 +49,13 @@ module.exports = {
     }
   }
 }
+
+interface Installation {
+  account: {
+    login: string
+  }
+}
+
+interface ExpressResponseWithDuration extends express.Response {
```

Comment:
Can this be added directly to the Express.Response interface? Similar to this? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-session/index.d.ts#L17-L20

Reply:
It could, but it seems simpler this way to me since we aren't exporting it. You're more familiar with types though. What benefits do you think it would add?